kythe/cxx/verifier/verifier.cc

```diff
@@ -681,24 +688,68 @@ bool Verifier::LoadInlineProtoFile(const std::string& file_data) {
       return false;
     }
   }
-  return parser_.ParseInlineRuleString(file_data, *kStandardIn,
-                                       "\\s*\\#\\-(.*)");
+  Symbol empty = symbol_table_.intern("");
+  return parser_.ParseInlineRuleString(file_data, *kStandardIn, empty, empty,
+                                       empty, "\\s*\\#\\-(.*)");
 }
 
 bool Verifier::LoadInlineRuleFile(const std::string& filename) {
-  bool parsed = parser_.ParseInlineRuleFile(filename, *goal_comment_regex_);
-  if (!parsed) {
+  int fd = ::open(filename.c_str(), 0);
+  if (fd < 0) {
+    LOG(ERROR) << "Can't open " << filename;
     return false;
   }
-  return true;
+  struct stat fd_stat;
+  if (::fstat(fd, &fd_stat) < 0) {
+    LOG(ERROR) << "Can't stat " << filename;
+    return false;
+  }
+  std::string content;
+  content.resize(fd_stat.st_size);
+  if (::read(fd, const_cast<char*>(content.data()), fd_stat.st_size) !=
+      fd_stat.st_size) {
+    LOG(ERROR) << "Can't read " << filename;
+    return false;
+  }
+  ::close(fd);
```

Comment:
this should likely use kythe/cxx/indexer/cxx/scope_guard.h (or that file should be moved to common, there's nothing indexer specific about it).